spacy/lookups.py

```diff
@@ -143,15 +157,88 @@ def from_dict(cls, data, name=None):
         self.update(data)
         return self
 
-    def __init__(self, name=None):
+class Table(OrderedDict):
```

Comment:
The `Table` class above can be removed, right?

Reply:
Ah, yes, that can go - this and the duplicated methods are leftovers from a bad merge. Thought I'd gotten rid of them...